.travis.yml

```diff
@@ -33,5 +34,5 @@ script:
     yarn start-func-test-server;
   fi
 - if [ $TOXENV ]; then tox; fi
-- if [ $TASK ]; then yarn $TASK; fi
+- if [ $TASK ]; then yarn run $TASK; fi
```

Comment:
I don't think `yarn run` is needed; won't `yarn test` and `yarn run test` do the same thing?